src/style.css

```diff
@@ -1 +1,46 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
According to the checklist, you should avoid using tag selectors for styling. In this case, you are applying styles to the `body` tag. It is recommended to use a class instead. For example, you could add a class `body` to your body tag and then apply these styles to that class. This will make your CSS more modular and easier to maintain.